src/core/best-practices.md

```diff
@@ -0,0 +1,52 @@
+---
+permalink: /core/best-practices
+---
+
+# Best Practices
+*Tips for conducting eye-tracking experiments with Pupil Core.*
+
+
+## Calibrating
+
+### Calibrate with Points That Resemble the Experimental Condition
+If your stimuli are at a fixed distance to the participant, make sure to place calibration points only at this distance. If the distance of your stimuli varies, place calibration points at varying distances. Make sure that your calibration points cover the outermost bounds of your gaze. Choose a calibration background color and brightness that matches your stimuli.
+
+### Validate Your Calibration
+Make a validation after the calibration to ensure that everything went fine. If the validation reports a high error, you need to perform another calibration. Make sure to use different points for validation than you used for calibration.
+
+
+## Avoiding Slippage
+Slippage is the effect of the headset slipping on the participant’s head. This is almost inevitable if the participant moves its head or its facial muscles. Slippage will lead to reduced accuracy and can accumulate over time.
+
+### Split Your Experiment into Blocks
+If your experiment allows for small breaks in between, you should split it into blocks. For every block you should recalibrate (and validate) at the beginning to reset any accumulated slippage errors.
+
+### Choose the Right Pipeline
+The 2D detection and mapping pipeline has the best accuracy in perfect conditions (< 1° visual error). However, it is very sensitive to slippage. Chose the 2D pipeline if:
+- Your participants do not need to move their heads.
+- You can have very short blocks.
+- You want the best possible accuracy.
+
+The 3D pipeline will adapt itself over time to compensate for slippage errors. The tradeoff is that the best accuracy in perfect conditions is slightly below the accuracy of the 2D pipeline (1.5°–2.0° visual error). Note that slippage will still have an effect on very long recordings. Chose the 3D pipeline if:
+- Your participants need to move their heads.
+- You cannot split your experiment into small blocks.
+- You are fine with good accuracy.
+
+
+## Record Everything
+You do not have to make fixed decisions for an experiment, as Pupil Player allows for running pupil detection and gaze mapping again with different settings in an offline mode. In order for this to work best, you should record all the data that is needed to run different offline algorithms. Here are the steps to include in your recording:
+1. Start the recording.
+1. Ask the participant to slowly look around with the eyes for a couple of seconds to sample different gaze angles. This will allow the 3D model to stabilize at the start. Do this even if you are not using the 3D pipeline to allow for comparison in offline mode.
```

Comment:
```suggestion
1. Ask the participant to slowly look around while keeping the head stable for a couple of seconds to sample different gaze angles. This will allow the 3D model to stabilize at the start. Do this even if you are not using the 3D pipeline to allow for comparison in offline mode.
```